google/cloud/aiplatform/training_jobs.py

```diff
@@ -408,6 +474,37 @@ def _create_input_data_config(
                     key=predefined_split_column_name
                 )
 
+            # Create timestamp split spec
+            timestamp_split = None
+            if timestamp_split_column_name:
+                if dataset._gca_resource.metadata_schema_uri not in (
+                    schema.dataset.metadata.tabular,
+                    schema.dataset.metadata.time_series,
+                ):
+                    raise ValueError(
+                        "A timestamp split may only be used with a tabular or time series Dataset"
+                    )
+
+                timestamp_split = gca_training_pipeline.TimestampSplit(
+                    training_fraction=training_fraction_split,
+                    validation_fraction=validation_fraction_split,
+                    test_fraction=test_fraction_split,
+                    key=timestamp_split_column_name,
+                )
+
+            if all(
+                [
+                    filter_split is None,
+                    predefined_split is None,
+                    timestamp_split is None,
+                ]
+            ):
+                fraction_split = gca_training_pipeline.FractionSplit(
+                    training_fraction=training_fraction_split,
+                    validation_fraction=validation_fraction_split,
+                    test_fraction=test_fraction_split,
+                )
+
```

Comment:
I plan on writing a couple more tests to check the behaviour when multiple types of splits are passed. But before I do that, I'd like y'all's thoughts on whether this is what we'd want.